CompanyEmployees.Infrastructure.Presentation/Controllers/AttendanceController.cs

```diff
@@ -0,0 +1,34 @@
+using CompanyEmployees.Application.Commands;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Shared.DataTransferObjects;
+
+namespace CompanyEmployees.Infrastructure.Presentation.Controllers;
```

Comment:
The semicolon after the namespace declaration is incorrect. Remove the semicolon to follow standard C# syntax for namespace declarations.

**Suggested change**
```suggestion
namespace CompanyEmployees.Infrastructure.Presentation.Controllers
```